lib/livebook/intellisense.ex

```diff
@@ -516,11 +516,12 @@ defmodule Livebook.Intellisense do
       end
 
     is_otp? =
-      case :code.which(app || module) do
-        :preloaded -> true
-        [_ | _] = path -> List.starts_with?(path, :code.lib_dir())
-        _ -> false
-      end
+      app == :erts or
```

Comment:
@josevalim in iex

```
Application.get_application(:atomics)
#=> nil
```

in iex -S mix

```
Application.get_application(:atomics)
# => :erts
```

Likely a result of https://github.com/elixir-lang/elixir/pull/12642, expected?

Reply:
Fixed!

Reply:
Perfect, thanks! I commented out the failing test and left a TODO, so we can marge the fix :)